sdk/dataprotection/Microsoft.Azure.Management.DataProtection/src/Microsoft.Azure.Management.DataProtection.Backup.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.Management.DataProtection.Backup</PackageId>
+		<Description>Provides developers with libraries for data protection under Azure Resource manager.</Description>
+		<Version>4.1.4-preview</Version>
```

Comment:
If this is the first time this package is being added, why is the version 4.1.4-preview?

The version should be 1.0.0-beta.1

Reply:
Changed the version